geomeppy/view_geometry.py

```diff
@@ -79,6 +79,27 @@ def view_polygons(polygons):
 
     plt.show()
 
+	
+def _get_geometry_rules(idf):
+    """Get the geometry rules from the IDF
```

Comment:
Add a ".", and bring the `"""` up to the same line. Nitpicky, but will keep `black` happy. In fact, better to just run `black` on the code and it will fix any issues.

`black -l 88 geomeppy tests docs setup.py release.py`

Reply:
Wow, `black` is pretty great!